internal/cli/atlas/atlas.go

```diff
@@ -100,5 +108,25 @@ func Builder() *cobra.Command {
 		accesslogs.Builder(),
 	)
 
+	updateHelpString(cmd, deprecatedMessage)
 	return cmd
 }
+
+// updateHelpString updates the help template to include the deprecated message.
+func updateHelpString(cmd *cobra.Command, deprecatedMessage string) {
+	for _, c := range cmd.Commands() {
+		c.SetHelpTemplate(fmt.Sprintf("%s \n%s", deprecatedMessage, c.HelpTemplate()))
+	}
+}
```

Comment:
this allows us to show the deprecated message when running the --help flags with mongocli atlas commands
Example:

```
./bin/mongocli atlas clusters --help
⚠ There’s a new, dedicated Atlas CLI available for Atlas users.  Install the Atlas CLI to enjoy the same capabilities and keep getting new features. Run brew install mongodb-atlas or visit https://dochub.mongodb.org/core/migrate-to-atlas-cli. Atlas commands for MongoCLI are now deprecated, but they will keep receiving support for 12 months (until April 30, 2023).


The clusters command provides access to your cluster configurations. You can create, edit, and delete clusters.
```